gitarro.rb

```diff
@@ -8,7 +8,14 @@
 require_relative 'lib/gitarro/backend'
 
 b = Backend.new
-prs = b.open_prs
+prs = b.open_newer_prs
+
+# Exit without errors if no PRs were found
+if prs.empty?
```

Comment:
even if rubocop will complain, we woulo d be good move this function to a class to backend.rb.
I would put it in the general class, or we could create a module which is included by the main class of backend.

Reply:
Ok, maybe we should call it inside Backend.open_newer_prs() and exit if the array is empty?

Reply:
Yo